### DISCUSSION TITLE
Primevue Verona Theme scss not converted in css

### DISCUSSION THREAD

**Discussion by phelp** [primefaces/community#325] [PrimeVue] [answered]
Hi there,

when i run npm run build in my project the dist folder is created with files also css files but ion the folder layout and theme there are still all scss files included. The href from index.html generated links to the css files.
href="/theme/indigo/theme-light.css">
href="/layout/css/layout-white.css">

anybody any ideas?

regards
Steffen

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

You need to run this sass command:

`sass --update public/layout/:public/layout/`
